src/components/Avatar/Avatar.tsx

```diff
@@ -125,34 +114,37 @@ export function Avatar({
   // Use `dominant-baseline: central` instead of `dy` when Edge supports it.
   const verticalOffset = '0.35em';
 
-  const initialsMarkup =
-    initials && !hasImage ? (
-      <span className={styles.Initials}>
-        <svg className={styles.Svg} viewBox="0 0 48 48">
-          <text
-            x="50%"
-            y="50%"
-            dy={verticalOffset}
-            fill="currentColor"
-            fontSize="26"
-            textAnchor="middle"
-          >
-            {initials}
-          </text>
-        </svg>
-      </span>
-    ) : null;
+  const svgMarkup = !hasImage ? (
+    <span className={styles.Initials}>
+      <svg className={styles.Svg} viewBox="0 0 40 40">
+        {avatarBody()}
+      </svg>
+    </span>
+  ) : null;
 
   return (
     <span aria-label={label} role="img" className={className}>
-      {initialsMarkup}
+      {svgMarkup}
       {imageMarkUp}
     </span>
   );
-}
 
-function customerPlaceholder(name?: string) {
-  return name
-    ? AVATAR_IMAGES[name.charCodeAt(0) % AVATAR_IMAGES.length]
-    : AVATAR_IMAGES[0];
+  function avatarBody() {
```

Comment:
Personal preference but I'm not a fan of having functions declared inside components unless needed, it seems like splitting this out into a separate function would be fairly simple. Then you could call it `AvatarBody` and  above you could replace` {avatarBody()}` with `<AvatarBody /`>